content/inside-rust/leadership-council-update@9.md

```diff
@@ -0,0 +1,101 @@
+- In response to a query regarding responsibility for stable versus unstable support in stable releases, we clarified that this generally falls under the Lang team's purview, with the Compiler team primarily responsible for implementation. [#180](https://github.com/rust-lang/leadership-council/issues/180)
+- We agreed to delegate decisions regarding GSoC mentor rewards to the Mentorship team. [#232](https://github.com/rust-lang/leadership-council/issues/232)
+- We considered a request to establish a Project Representative position but decided against it for the time being. [#226](https://github.com/rust-lang/leadership-council/issues/226)
+
+## Ongoing work
+
+Several projects have seen significant discussion since our last update but have not yet reached a conclusion.
+
+### Foundation
+
+We are excited about the development of the [Rust Foundation Maintainers Fund](https://rustfoundation.org/media/announcing-the-rust-foundation-maintainers-fund/). We are working closely with the Foundation to guide how this program will fund Rust Project maintainers. A working group is forming to define the usage of these funds ([#248](https://github.com/rust-lang/leadership-council/pull/248)). We expect to have an approved plan and policy in place in January 2026. [#242](https://github.com/rust-lang/leadership-council/issues/242)
+
+David Wood has proposed a clarification of Project Director responsibilities, which we hope will improve understanding and communication between the Council, the Directors, and the Foundation. [#250](https://github.com/rust-lang/leadership-council/pull/250)
+
+### Additional items
+
+- We are following up on a request to add Graydon Hoare to the alumni section of the website.
+[#231](https://github.com/rust-lang/leadership-council/issues/231)
+- Regarding a request to link the community Discord from our website, we have deferred the decision to the website management team. [#219](https://github.com/rust-lang/leadership-council/issues/219)
+- We received invitations to apply for funding from the Rust Ecosystem Fund. [#240](https://github.com/rust-lang/leadership-council/issues/240)
+
+## Following our work
+
+As you can see from the many links above, the work of the Council happens mostly in public on GitHub issues posted in [our repository](https://github.com/rust-lang/leadership-council/issues). The items on our meeting agendas are drawn from these. After discussing an item, we summarize that discussion and any shared [rationales](https://aturon.github.io/tech/2018/05/25/listening-part-1/) on the issue. When we make decisions, we propose on the issue a "final comment period" (FCP), and as with all FCPs in the Project, we're interested in and review any feedback that people have before or during this final comment period.
+
+To follow our work in real time, watch our repository.
```

Comment:
Watching the repo is IMO not all that useful, although it definitely helps. I would mention the new Zulip stream where Tomáš posts meeting minute summaries, I think that works well for following our work.

Reply:
https://rust-lang.zulipchat.com/#narrow/channel/392734-council/topic/Meeting.20minutes.20.26.20summaries/with/561198432